src/dataplane-protocol/src/batch.rs

```diff
@@ -111,7 +111,25 @@ where
     }
 }
 
-impl Batch<DefaultBatchRecords> {
+impl DefaultBatch {
+    // TODO write test
```

Comment:
Can close PR once this test is completed

Reply:
Added test showing batch behavior is the same as individual-adding behavior